purchase_requisition_blanket_order_report/report/report_purchase_requisition.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template
+        id="report_purchaserequisitions"
+        inherit_id="purchase_requisition.report_purchaserequisitions"
+    >
+        <xpath expr="//div[hasclass('page')]/h2" position="replace">
```

Comment:
Looks like you need a better option than using `replace`, according to the pre-commit settings.

Reply:
For this case, IMO the replace makes sense (since it's still possible to target any element in the replaced node through XPath). So pre-commit be damned. :smiling_imp:

Reply:
But, if another module extended this node before this module, his changes would be lost, no ?

Reply:
In that case yes, but it's not even sure the changes from the other module would be applied before the changes from this one. Anyway, if both modules have to co exist on the same instance and there is an issue appearing, then it can be resolved by inheriting from this module in the other module. (and the risk is quite small considering the change that is here).